samples/winapi_drivelist/main.c

```diff
@@ -58,9 +58,21 @@ int main(void)
     }
     debugPrint("\ndone");
 
-    while(1) {
-        Sleep(2000);
+    Sleep(20000);
+    
+    ret = nxUnmountDrive('C');
+    if (!ret) { // if there was an error while unmounting (nxUnmountDrive() returns false)
```

Comment:
We don't do these sort of inline-comments. Check surrounding code for how it should be done.